mcs/class/Mono.Debugger.Soft/Mono.Debugger.Soft/Connection.cs

```diff
@@ -792,10 +795,12 @@ class PacketReader {
 
 				// For reply packets
 				offset = 0;
-				ReadInt (); // length
+				var len = ReadInt (); // length
 				ReadInt (); // id
 				ReadByte (); // flags
 				ErrorCode = ReadShort ();
+				if (ErrorCode == (int)Mono.Debugger.Soft.ErrorCode.INVALID_ARGUMENT && len > offset)
```

Comment:
Do we need a protocol bump for this? @thaystg

Reply:
I think len > offset takes care of it, but it would be nicer to do it with a protocol bump.

Reply:
Agreed with @vargaz.

Reply:
I'm bumping the protocol but I still need to check with  len > offset  because I don't know if I inserted the error message everywhere that is needed.